src/register.js

```diff
@@ -0,0 +1,29 @@
+
+export function getProducts(products, code) {
+    for(let i = 0; i < products.length; i++) {
+        const product = products[i];
+        if(product.code === code) {
+            return product;
+        }
+    }
+    return null;
+} 
+
+export function lineTotes(thing, num) {
+    return Number((thing * num).toFixed(2));
+}
+
+
+export default function makesTotal(cart, products) {
+    let total = 0;
+        
+    for(let i = 0; i < cart.length; i++) {
+        const element = cart[i].code;
```

Comment:
re: naming - be more explicit/descriptive. "productCode" or something like that would work well here.